action.yml

```diff
@@ -30,7 +30,7 @@ runs:
     - name: Install dependencies
       shell: bash
       run: |
-        pip install mergify-cli
+        pip install "mergify-cli==2025.7.29.1"
```

Comment:
I don't think we should do that as it means we have to update it manually.
It's faster to fix mergify-cli